mlflow/cli.py

```diff
@@ -47,7 +47,7 @@ def cli():
                    "Databricks workspace specified in the default Databricks CLI profile. "
                    "See https://github.com/databricks/databricks-cli for more info on configuring "
                    "a Databricks CLI profile.")
-@click.option("--cluster-spec", metavar="FILE",
+@click.option("--cluster-spec", "-c", metavar="FILE",
```

Comment:
Like the short option style, but this is the only one that has it. A bit inconsistent.

Reply:
Many of the other options on this command also have short versions, don't they? Look above in the diff, in the folded-in part.